curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/65ca2d0625aa3a3201067f70.md

```diff
@@ -0,0 +1,168 @@
+---
+id: 65ca2d0625aa3a3201067f70
+title: Step 30
+challengeType: 0
+dashedName: step-30
+---
+
+# --description--
+
+In the callback function of your `forEach`, use the `el` parameter to access the `counts` object. If the element is not in `counts`, set it to `0`. Otherwise, increment the count for the element. 
```

Comment:
The first sentence is a little confusing and was mentioned in the original issue.

```suggestion
Inside the `array.forEach()` callback function, check if the current element is inside the `counts` object.  If the element can be found, update the value of `counts[el]` by `1`. Otherwise, assign the number `0` to `counts[el]`.
```